contracts/BondController.sol

```diff
@@ -268,6 +270,7 @@ contract BondController is IBondController, OwnableUpgradeable {
      */
     function getTrancheLetter(uint256 index, uint256 _trancheCount) internal pure returns (string memory) {
         bytes memory trancheLetters = bytes("ABCDEFGHIJKLMNOPQRSTUVWXY");
+        require(index < trancheLetters.length, "BondController: invalid tranche letter");
```

Comment:
What's the intended behavior for `index=26`?

Reply:
it fails

Reply:
it works